src/api/services/TutorService.ts

```diff
@@ -26,8 +26,8 @@ export class TutorService {
         return await this.repository.save(tutor);
     }
 
-    public async remove(id: any): Promise<void> {
-        await this.repository.remove(id);
+    public async delete(id: any): Promise<void> {
+        await this.repository.delete(id);
```

Comment:
@bigLucas 
The `DELETE` method is now working. The error was occurring because I changed the name of the _TypeORM_ function from `delete` to `remove`.
And I decided to change the name of our method to delete too, to keep a pattern.